mod/metrics/set/type_plus_right/metric_value/value.rb

```diff
@@ -8,7 +8,7 @@ def value_card
 end
 
 def value
-  content
+  item_names.join ", "
```

Comment:
we run this for all values?  interesting.  could it break on multiline free-text?